swarm/api/http/server.go

```diff
@@ -558,7 +563,7 @@ func (s *Server) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 	uri, err := api.Parse(strings.TrimLeft(r.URL.Path, "/"))
 	if err != nil {
 		s.logError("Invalid URI %q: %s", r.URL.Path, err)
-		http.Error(w, fmt.Sprintf("Invalid bzz URI: %s", err), http.StatusBadRequest)
+		ShowError(w, r, fmt.Sprintf("Invalid bzz URI: %s", err), http.StatusBadRequest)
```

Comment:
Does it make sense for this function to be a method on `Server` like the other ones (i.e. `s.Error(...)`)?